schema.graphql

```diff
@@ -168,6 +168,7 @@ type Juror @entity {
   anjMovements: [ANJMovement!] @derivedFrom(field: "juror")
   feeMovements: [FeeMovement!] @derivedFrom(field: "owner")
   treasuryBalances: [TreasuryBalance!] @derivedFrom(field: "owner")
+  claimedFees: [JurorSubscriptionFee!] @derivedFrom(field: "juror")
```

Comment:
Could we rename it to `claimedSubscriptionFees` ? Just to not create confusion with other fees in the frontend :D

Reply:
haha np